components/range-control/index.js

```diff
@@ -26,7 +26,7 @@ function RangeControl( {
 } ) {
 	const id = `inspector-range-control-${ instanceId }`;
 	const onChangeValue = ( event ) => onChange( Number( event.target.value ) );
-	const initialSliderValue = value === undefined ? initialPosition || '' : value;
+	const initialSliderValue = ! value && 0 !== value ? initialPosition || '' : value;
```

Comment:
I'm wondering if we shouldn't flip the check as follows:
```js
isNumber( value ) ? value : initialPosition || '',
```
Would that work?

Reply:
Code was updated to use `isFinite( value ) ? value : initialPosition || '';`.